Lsunae/FBMember/app/src/main/java/user/manage/fbmember/LoginActivity.kt

```diff
@@ -0,0 +1,42 @@
+package user.manage.fbmember
+
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import androidx.appcompat.app.AppCompatActivity
+import com.google.firebase.auth.FirebaseUser
+import kotlinx.android.synthetic.main.login.*
+import kotlinx.android.synthetic.main.register.*
+
+class LoginActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.login)
```

Comment:
```suggestion
        setContentView(R.layout.activity_login)
```
activity_pagename 형식으로 지어주세요. android layout  code convention

Reply:
수정했습니다.